### ISSUE TITLE
Undefined method 'dispatchBeforeLoadEvent' in REST API

### ISSUE THREAD

**Issue by ppi-buck** [pimcore/pimcore#3024]
# Bug Report
The REST API doesn't return a list of assets on **Search Asset** as documented here
https://pimcore.com/docs/5.x/Development_Documentation/Web_Services/index.html#page_Search-Assets 

### Expected behavior 
Get a list of assets matching the given criteria.
  
### Actual behavior  
API returns an error with success: false and the code below.   

### Steps to reproduce

- GET 
 https://demo-basic.pimcore.org/webservice/rest/asset-list?apikey=6e33d283712a0f76a3fa6d657e148d97ca5467da149b378cf9b664d81f3ba7c1   (api-key might be out of date when demo resets)
  
### Code example 
```
{
   "success": false,
   "msg": "Attempted to call an undefined method named "dispatchBeforeLoadEvent" of class 
   "Pimcore\Bundle\AdminBundle\Controller\Rest\Element\AssetController"."
}
```

**Comment by brusch** [comment]
Fixed already by 3c72c31086a97ab6f1fdf9407789547b465c8ed3

**Comment by ppi-buck** [comment]
Sorry, must have updated right before the new build was released.
Thanks!
